web_src/less/_base.less

```diff
@@ -541,6 +541,10 @@ code,
     margin: 0 6px;
   }
 
+  .commit-sha {
+    font-family: @monospaced-fonts, monospace;
```

Comment:
@silverwind I found ``@monospaced-fonts`` is not defined after merge master, Please have a look. Thanks
log:
```SHELL
ERROR in ./web_src/less/index.less
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/less-loader/dist/cjs.js):


  .commit-sha {
    font-family: @monospaced-fonts, monospace;
               ^
Variable @monospaced-fonts is undefined
      Error in /gitea_work/gitea/web_src/less/_base.less (line 544, column 17)
    at /gitea_work/gitea/node_modules/webpack/lib/NormalModule.js:316:20
    at /gitea_work/gitea/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /gitea_work/gitea/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at context.callback (/gitea_work/gitea/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.lessLoader (/gitea_work/gitea/node_modules/less-loader/dist/index.js:57:5)
 @ multi ./web_src/js/jquery.js ./web_src/fomantic/build/semantic.js ./web_src/js/index.js ./web_src/fomantic/build/semantic.css ./web_src/less/index.less index[4]
```

Reply:
You need to use `font-family: var(--fonts-monospace);` now.

Reply:
See suggestion below.